iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/function_ref.inl

```diff
@@ -69,6 +94,7 @@ template <class ReturnType, class... ArgTypes>
 inline ReturnType function_ref<ReturnType(ArgTypes...)>::operator()(ArgTypes... args) const noexcept
 {
     // Expect that a callable was assigned beforehand
+    // AXIVION Next Line AutosarC++19_03-M5.3.1 : 'nullptr' check shall be performed explicitly
```

Comment:
Could this be avoided by casting the string literal to bool instead? It is of not much importance as "Ensures" will be removed in the future.

Reply:
@MatthiasKillat Jep, see description of this PR. This topic is left open for you to address.